### PR TITLE
mo-shakib

### LINKED ISSUE #675: Mohammad Shakib
image

### PR DESCRIPTION
**Pull request by Mo-Shakib**
resolve #675

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1288,11 +1288,17 @@
...
+  },
   {
     "name": "Anas Khan",
     "githubUsername": "anaskhan28",
     "imageUrl": "https://user-images.githubusercontent.com/87796038/130655483-7fd4dc84-9f0d-412b-acac-68ab178aaf3c.png",
     "issueNumber": 666
+  },
+  {
+    "name": "Mohammad Shakib",
+    "githubUsername": "mo-shakib",
```
Comment (Panquesito7) [inline data.json:1300]:
```suggestion
    "githubUsername": "Mo-Shakib",
```

File: data.json [hunk 2]
```diff
@@ -1288,11 +1288,17 @@
...
+    "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
+    "issueNumber": 675
   }
-]
+]
```
Comment (Panquesito7) [inline data.json:1304]:
```suggestion
]

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Next time, please don't create another PR for the same thing :) instead, implement the changes in the same PR.

File: data.json (hunk 1 above)
Comment (Mo-Shakib) [reply data.json:1300]: my bad! actually, it's my very first time that creating PR :)

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 thank you for your contribution! 😄👍

**Pull request merged**